### ISSUE TITLE
Distorted Screen in OpenVR apps

### ISSUE THREAD

**Issue by DoubleBeeLive** [alvr-org/ALVR#367]
### I get **distorted**, skewed picture, its like the FOV is completely wrong. 

The problem happens only in two apps: Flight Simulator 2020 and City Car Driving. As FS2020 uses OpenVR, I suppose its somehow related.It works OK in the SteamVR menu, and all other games I played, but not those two. Those two, however, have _identical_ problems.
The Problem is pretty rare, i only found it once on the old ALVR github page:

[https://github.com/polygraphene/ALVR/issues/683](https://github.com/polygraphene/ALVR/issues/683)

and a random forum

[https://forum.scssoft.com/viewtopic.php?t=270083](https://forum.scssoft.com/viewtopic.php?t=270083)

**It is impossible to screenshot or record this problem**, and hard to describe, but it is pretty trippy. It is like the perspective is completely messed up.

There was a hint to change the IPD settings, but I tried  and it didnt really work.

#### Environement

3060ti, latest ALVR and SteamVR Beta

**Comment by DoubleBeeLive** [comment]
more clarification, if you want to know the lore: 

**tldr**: wanted a good driving sim years ago, bought a new RTX, still cant run it

~~long boi~~

a year ago I wanted to learn for my driving license in a simulator and searched for some driving games. Project Cars was a bomb, spent countless hours in it but there were no real-life rules so I needed a game with traffic rules in it. Sadly the only game that supported those was a ~~hot steaming pile of garbage of a~~ game called City Car Driving. I tried to play it on the screen, but after tens of hours  it was not immersive enough. With my old 1060 3gb i tried to launch the Oculus Menu, but it was terrible at max 7fps, so I simply gave up.
Year passed by, I try to connect my Ouclus Quest again to an unrealted game, but get "oculus has detected a problem with your graphics drivers" error, after connecting Oculus Link. I search for another opportunities to game on my Quest and MIRACLE, i found a much-better-then-tethered solution, that doesnt include 1) TP-Cast for 250€ nor 2) HTC Wireless adapter for even more €. 

_ALVR, i thought i was dreaming._ My **1060 could launch VR games!** And even play them at _stable framerates_, wirelessly! I was very happy until I startd City Car Driving.
Que my dissapoinment: it had a stable 40 fps, but stuttered very much, that it felt like 4 fps. Another big problem was the distortion. What i saw in VR was skewed and behaved really strange. I tried it all: reinstalled everything, drivers, Steamvr. I even bought the game, thinking this would fix the issue, but no chance. I came to the conclusion, that it was a game specific issue and slowly gave up, blaming 1) CCD 2) my old poor 3GB 1060

Time flied by, Nvidia released some cards, out of all releases i managed to only by the worst of them, an 3060ti. But I am really in love with that card, not gonna lie. Everything flies like a jet, DLSS is the future. Except for City Car Driving with it strange behaviour.

Imagine my suprise, as i saw exactly the same issue as in City Car Driving in FS2020. 

But now i know, it may have the same culprit, because ALVR is the only thing, making the games related to each other

**Comment by zarik5** [comment]
You can try ALVR ev11. It had a settings made especially to solve this problem. In Other tab, something like "warp fix".

**Comment by DoubleBeeLive** [comment]
> You can try ALVR ev11. It had a settings made especially to solve this problem. In Other tab, something like "warp fix".

tried it, nothing changed
